<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>952</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="testImg">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>550</y>
      <width>168</width>
      <height>16</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Wallpaper_Image</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QFrame" name="boxNav">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>11</y>
      <width>152</width>
      <height>122</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="gameButton">
       <property name="minimumSize">
        <size>
         <width>128</width>
         <height>28</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Маслина</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="programButton">
       <property name="minimumSize">
        <size>
         <width>128</width>
         <height>28</height>
        </size>
       </property>
       <property name="text">
        <string>Программульки</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="settingButton">
       <property name="minimumSize">
        <size>
         <width>128</width>
         <height>28</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Настройки</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="boxMenu">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>10</y>
      <width>168</width>
      <height>281</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="fullScreenButton">
       <property name="minimumSize">
        <size>
         <width>144</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>FullScreen</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="minimumSize">
        <size>
         <width>144</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="fileManagerButton">
       <property name="minimumSize">
        <size>
         <width>144</width>
         <height>32</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>144</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>FileManager</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="dirDlgButton">
       <property name="minimumSize">
        <size>
         <width>144</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>Choose Folder</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearButton">
       <property name="minimumSize">
        <size>
         <width>144</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="trashButton">
       <property name="minimumSize">
        <size>
         <width>144</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>Recycle.Bin</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="updateButton">
       <property name="minimumSize">
        <size>
         <width>144</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>Update Form</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="movePanel" native="true">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>-40</y>
      <width>521</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 25px;
border-color: white;
border-width: 1px;
border-style: inset;
background: #303030;</string>
    </property>
   </widget>
   <widget class="QFrame" name="signatureBox">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>540</y>
      <width>109</width>
      <height>61</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="versionLabel">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="midLineWidth">
        <number>1</number>
       </property>
       <property name="text">
        <string>&lt;p&gt;version 0.0.0.0&lt;/p&gt;</string>
       </property>
       <property name="textFormat">
        <enum>Qt::MarkdownText</enum>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <italic>true</italic>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>by Zealot</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <property name="indent">
        <number>1</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>TODO</string>
    </property>
   </widget>
   <widget class="QFrame" name="workspacePanel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>751</width>
      <height>401</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>12</number>
     </property>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
